Session5/Assignment5/EmployeeManagementTests/Controllers/PromotionsControllerTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EmployeeManagement.Model;
+using EmployeeManagement.Services;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+
+namespace EmployeeManagement.Controllers.Tests
+{
+    public class PromotionsControllerTests
+    {
+        [Fact()]
+        public void CreatePromotion_InternalEmployee_Returns_Created()
+        {
+            // Arrange
+            var _stubEmployeeService = new Mock<IEmployeeService>();
+            var _stubPromotionService = new Mock<IPromotionService>();
```

Comment:
PromotionService test double should be Mock. Interaction is changing state - PromoteInternalEmployeeAsync

Reply:
Updated, thanks sir!